### PR TITLE
Add Drupal core version and php version to the user agent string

### LINKED ISSUE #556: Add Drupal core version to the user agent string
## Description
When calls are made to Apigee APIs, the user agent string sends info about the Apigee Edge version and PHP Client version:

`Apigee Edge DevPortal 8.x-1.10 (Apigee Edge PHP Client 2.0.3)`

This task is to add the Drupal core version along with this information:

`Drupal Core 8.19, Apigee Edge DevPortal 8.x-1.10 (Apigee Edge PHP Client 2.0.3)`

### PR DESCRIPTION
**Pull request by shishir-intelli**
Closes #556 

Added Drupal core version, Apigee_edge module and m10 module versions to the user agent string.
Related Client PHP PR - https://github.com/apigee/apigee-client-php/pull/171

### CODE REVIEW

File: src/SDKConnector.php [hunk 1]
```diff
@@ -250,13 +250,23 @@ private function buildCredentials(KeyInterface $key): CredentialsInterface {
...
-        $module_info['version'] = '8.x-1.x-dev';
+
+      $edge_module_info = $this->infoParser->parse($this->moduleHandler->getModule('apigee_edge')->getPathname());
+      if (!isset($edge_module_info['version'])) {
+        $edge_module_info['version'] = '8.x-1.x-dev';
+      }
+      $edge_module_info = $edge_module_info['name'] . '/' . $edge_module_info['version'];
+
+      $m10_module_info = '';
+      if ($this->moduleHandler->moduleExists('apigee_m10n')) {
```
Comment (mxr576) [inline src/SDKConnector.php]: This should be added by the apigee_m10n module to the user agent string, the dependency must not know about any its dependants.

**Review by mxr576 — CHANGES_REQUESTED** [review]
Nitpick: Commit/PR title does not have to end with a full stop.

File: src/SDKConnector.php (hunk 1 above)
Comment (mxr576) [reply src/SDKConnector.php]: @shishir-intelli  @phdhiren What do you think about my comment?

Comment (phdhiren) [reply src/SDKConnector.php]: Let's implement via custom hook and let apigee_m10n module extend this.

Comment (shishir-intelli) [reply src/SDKConnector.php]: Thanks @mxr576,
That makes sense to me.

**Review by phdhiren — APPROVED** [review]
LGTM

**Pull request merged**